src/trigrams.py

```diff
@@ -0,0 +1,91 @@
+import random
+import io
+import os
+import sys
+
+
+dirpath = os.path.dirname(os.path.abspath(__file__))
+source_file = os.path.join(dirpath, 'body_text.txt')
+
+
+def main(input_body=source_file, n="200"):
+    """Main calls the other functions and defines word count param."""
+    if os.path.isfile(input_body):
+        if os.stat(input_body).st_size >= 1:
```

Comment:
This can probably be combined with the previous line to check for both conditions returning a common bad-input message for both.